dynamic_programming/Minimum_cost_path.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+
+def minimum_cost_path(matrix: list[list[int]]) -> int:
+    """
+    Find the minimum cost traced by all possible paths from top left to bottom right in
```

Comment:
Adding a few more doctests of cases in which paths are negative integers would be nice